docs/grid/custom_field_type.md

```diff
@@ -1,11 +1,11 @@
-Creating custom Field type
+Creating a custom Field type
```

Comment:
Same question about file header as in previous https://github.com/Sylius/Stack/pull/180#discussion_r1888383950

```suggestion
Creating a custom Field Type
```